.travis.yml

```diff
@@ -1,47 +1,72 @@
-
+sudo: false
+os:
+  - linux
+  - osx
```

Comment:
I'd like it better if `language` remained the first one, because for Travis, it's the most important (it sets the defaults for all the other things, picks the base image, etc)?

The `sudo: false` could be immediately after, since it's pretty important as well, but `os` should definitely be together with `compiler` and `env`, because these three are the ones which create the matrix of builds.

And I'd like a newline between top-level things. :smiley: